modules/shutter_page/storage_container.tf

```diff
@@ -1,6 +0,0 @@
-resource "azurerm_storage_container" "containers" {
```

Comment:
how is this happening now?

Reply:
This is happening by default now when you enable the static website